client/docker/README.md

```diff
@@ -0,0 +1,40 @@
+# Install 311 Server locally for development
+
+This doc in intended for client side developers working on the 311 Data project who want an easy way to run a local copy of the server. It downloads images for whatever code is in the ```dev``` branch in GitHub.
+
+## Introduction and Prerequisites
+
+These instructions assume only that Docker is installed. They also assume the steps are being followed on a Mac, but it should be easy enough to follow for other platforms.
+
+All of the 311 Server components (e.g. Postgres, Redis) are downloaded and run as Docker containers. This wouldn't be the best configuration for a production system, but it is more than adequate for development.
+
+The 4 included images will need about 1 GB of space with an additional 1 GB or so needed for the data.
+
+## Initialization Script
```

Comment:
What do you think about swapping the positions of Initialization Script and Installation Instructions sections? Devs (me) who don't read through the whole README first will see instructions first then an explanation of what the init script does.

Reply:
I was confused if i had to create the Initialization script myself. Would be good to just have the install instructions to make it simpler but otherwise everything works when executing the bash script. Awesome work!